Misc/NEWS.d/next/C API/2021-09-19-17-18-25.bpo-44687.3fqDRC.rst

```diff
@@ -0,0 +1 @@
+Added additional checks to :c:macro:`CHECK_CLOSED` macro to fix unintended closing file error when peeking a BufferedReader object
```

Comment:
```suggestion
:meth:`BufferedReader.peek` no longer raises :exc:`ValueError` when the entire file has already been buffered.
```

Would still like to see this be more helpful for _users_ (who aren't aware of the macro at all).

Reply:
ah yes that is definitely better, thanks for the suggestion